scripts/push-vellum-workflows.ts

```diff
@@ -516,13 +516,27 @@ class VellumWorkflowPusher {
         return;
       }
 
-      const changedFiles = execSync(
-        "git diff --name-only HEAD~1 HEAD -- vellum/services/",
-        {
-          encoding: "utf8",
-          cwd: process.cwd(),
+      let changedFiles = "";
+      try {
+        changedFiles = execSync(
+          "git diff --name-only HEAD~1 HEAD -- vellum/services/",
```

Comment:
Instead of doing a git diff at all, we should use the GitHub api to get the changed files in the latest commit on a branch